docs/api/server/runtimes/common.md

```diff
@@ -0,0 +1,1120 @@
+---
+title: Common
+order: 1
+---
+
+# Server / Runtimes / Common
+
+## HTTP Helpers
+
+### `json`
+
+This is a shortcut for creating `application/json` responses. It assumes you are using `utf-8` encoding.
+
+```ts lines=[2,6]
+import type { LoaderFunction } from "remix";
+import { json } from "remix";
```

Comment:
I think it would be better to have specific imports instead of the global package, as we want to move away from it

```suggestion
import type { LoaderFunction } from "@remix-run/<runtime>";
import { json } from "@remix-run/<runtime>";
```

Reply:
Yes, that is the purpose of this PR. I'm in progress of doing the import conversion. Will mark PR as ready to review once all imports have been converted.